jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/CopyMessageAction.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.actions
+
+import com.intellij.icons.AllIcons
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.ide.CopyPasteManager
+import com.intellij.openapi.project.DumbAwareAction
+import com.intellij.ui.table.TableView
+import software.amazon.awssdk.services.sqs.model.Message
+import software.aws.toolkits.resources.message
+import java.awt.datatransfer.StringSelection
+
+class CopyMessageAction(private val table: TableView<Message>) : DumbAwareAction(message("sqs.copy.message", 1), null, AllIcons.Actions.Copy) {
+    override fun update(e: AnActionEvent) {
```

Comment:
should we tie these actions to the standard shortcuts? So if they hit `cmd/ctrl + c` it will copy?

Reply:
Yes, let me find the other place we do that